src/test/resources/db/scripts/schema.sql

```diff
@@ -1,74 +1,108 @@
-
-    alter table if exists affected 
-       add constraint fk_affected_parent_id_disturbance_id 
-       foreign key (parent_id) 
-       references disturbance (id);
+
+create table affected (
+                          id bigint not null auto_increment,
+                          parent_id bigint not null,
+                          reference varchar(512),
+                          coordinates varchar(255),
+                          facility_id varchar(255),
+                          party_id varchar(255),
+                          primary key (id)
+) engine=InnoDB;
+
+create table disturbance (
+                             deleted bit,
+                             created datetime(6),
+                             id bigint not null auto_increment,
+                             planned_start_date datetime(6),
+                             planned_stop_date datetime(6),
+                             updated datetime(6),
+                             description varchar(8192) not null,
+                             category varchar(255) not null,
+                             disturbance_id varchar(255) not null,
+                             status varchar(255) not null,
+                             title varchar(255),
+                             primary key (id)
+) engine=InnoDB;
+
+create table disturbance_feedback (
+                                      created datetime(6),
+                                      id bigint not null auto_increment,
+                                      category varchar(255) not null,
+                                      disturbance_id varchar(255) not null,
+                                      party_id varchar(255) not null,
+                                      primary key (id)
+) engine=InnoDB;
+
+create table disturbance_feedback_history (
+                                              created datetime(6),
+                                              id bigint not null auto_increment,
+                                              category varchar(255) not null,
+                                              disturbance_id varchar(255) not null,
+                                              party_id varchar(255) not null,
+                                              status varchar(255) not null,
+                                              primary key (id)
+) engine=InnoDB;
+
+create table feedback (
+                          created datetime(6),
+                          id bigint not null auto_increment,
+                          party_id varchar(255) not null,
+                          primary key (id)
+) engine=InnoDB;
+
+create table opt_out_settings (
+                                  id bigint not null auto_increment,
+                                  subscription_id bigint,
+                                  category enum ('COMMUNICATION','DISTRICT_COOLING','DISTRICT_HEATING','ELECTRICITY','ELECTRICITY_TRADE','WASTE_MANAGEMENT','WATER') not null,
```

Comment:
Is it possible to make this `category varchar(255) not null` ?

Reply:
Yes!